docs/components/pages/home/home.component.tsx

```diff
@@ -0,0 +1,236 @@
+import React, { useCallback, useEffect, useState } from "react";
+import { memoCacheKey } from "@swyg/memo";
+import styles from './home.module.scss';
+import { CacheMonitor } from "../../cache-monitor/cache-monitor.componen";
+import { CacheItem } from "@swyg/memo";
+import { getFriendlyTimeLabel } from "../../../utils/dateFormatting";
+
+function sleep(wait: number) {
+    return new Promise(resolve => setTimeout(resolve, wait));
+}
+
+async function fakeFetch(param1: string, param2: string) {
+    await sleep(2000 + Math.random() * 2000);
+
+    return new Promise((resolve) => {
+        resolve(`${ param1 } | ${ param2 }`);
+    });  
+}
+
+const memoFakeFetch = memoCacheKey(fakeFetch, {
+    keepPromises: true,
+    slots: 8,  
+    maxAge: 10000,
+});
+
+const cacheProxy = memoFakeFetch.cache;
+
+function getObservableCache() {
```

Comment:
@vincentropy This is the bit that I was telling you could be turn into a library (e.g. `getObservableObject()`). The idea is that you get a Proxy wrapper around any object you want to observe, but you can just pass it around and subscribe to changes later (e.g. `myProxy.subscribe(() => ...)`.

Reply:
🤔 Interesting idea. Do you see this used to build functionality of more for logging/diagnostics?

Reply:
Potentially yes. This functionality is quite simple, it only provides a way for you to wrap an object in a Proxy but subscribe to changes later, as opposed to creating the Proxy with the hardcoded code to handle the different calls in here.

I haven't thought much about it, but that could be mentioned in the docs, as I see how this could be use for that.